2.cellprofiler_processing/example_cp.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CellProfiler Segmentation and Feature Extraction of CFReT data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pathlib\n",
+    "\n",
+    "import importlib\n",
+    "CPutils = importlib.import_module(\"cputils\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set paths for CellProfiler\n",
+    "\n",
+    "The paths for the command line command can be set here. These must be `absolute paths`, so these will have to be changed from what I have set below for my local machine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# must be an absolute path to the pipeline \n",
+    "path_to_pipeline = \"/home/jenna/CFReT_data/2.cellprofiler_processing/CellProfiler_input/CFReT_project.cpproj\"\n",
+    "\n",
+    "# must be relative path for the output directory\n",
+    "path_to_output = \"/home/jenna/CFReT_data/2.cellprofiler_processing/CellProfiler_output\"\n",
+    "\n",
+    "# must be relative path to the images\n",
+    "path_to_images_plate1 = \"/home/jenna/CFReT_data/1.preprocessing-data/IC_Corrected_Images/localhost220512140003_KK22-05-198\"\n",
+    "path_to_images_plate2 = \"/home/jenna/CFReT_data/1.preprocessing-data/IC_Corrected_Images/localhost220513100001_KK22-05-198_FactinAdjusted\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run CellProfiler through command line\n",
+    "\n",
+    "Below is an example of the code used in terminal to run CellProfiler on one of the plates. Since there is already a file there, it will show that it can't run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Per_Image, Per_Cells, Per_Cytoplasm and Per_Nuclei tables already in database and overwrite not allowed. Exiting\n"
```

Comment:
Is there a way to force overwriting? Would you want to allow for this, or is how you have it ok?

Reply:
You can if you put the CellProfiler .cppipe file into the GUI. I am fine with it not overwriting since you should only need to run it once... to each their own lol. I will add that to the README.